### PR TITLE
[SECURESIGN-232] [SECURESIGN-233] [SECURESIGN-234] | Fulcio custom pipeline config

### PR DESCRIPTION
**Pull request by tommyd450**

### CODE REVIEW

**Review by JasonPowr — APPROVED** [review]
/lgtm

**Comment by Gregory-Pereira** [comment]
Confused why were not using the golang opnshift builder image but if registry go-tools image works thats fine.
/approve
/lgtm

File: Dockerfile [hunk 1]
```diff
@@ -13,31 +13,22 @@
...
 
 WORKDIR $APP_ROOT/src/
 ADD go.mod go.sum $APP_ROOT/src/
-RUN go mod download
-
 # Add source code
 ADD ./ $APP_ROOT/src/
 
-RUN go build -o server main.go
-RUN CGO_ENABLED=1 go build -gcflags "all=-N -l" -o server_debug main.go
```
Comment (Gregory-Pereira) [inline Dockerfile:28]: Why are we removing the dubgging binary in the build process?

**Comment by tommyd450** [comment]
> Confused why were not using the golang opnshift builder image but if registry go-tools image works thats fine. /approve /lgtm

I hadnt even noticed I havent updated the builder!

**Comment by lance** [comment]
> > Confused why were not using the golang opnshift builder image but if registry go-tools image works thats fine. /approve /lgtm
> 
> @Gregory-Pereira I hadnt even noticed I havent updated the builder!

If we can stick to the older go version without any build problems, then we should. Occasional a repo will use a 1.21 feature and that's where we get stuck and have to revert to the openshift brew image. I don't think it's necessary in this case, and pulling from registry.redhat.io is preferred.

File: Dockerfile (hunk 1 above)
Comment (lance) [reply Dockerfile:28]: Do we need it? There is no Red Hat documentation for it, and it wasn't used in any way in TP1. I think if we don't need to include it, let's don't just to keep the surface area as small as possible.

**Comment by lance** [comment]
@Gregory-Pereira I'm going to go ahead and merge this. If you want to pursue including the debug binary, we can do that in another PR.

**Pull request merged**